common/src/main/java/net/william278/husksync/config/Settings.java

```diff
@@ -99,6 +99,9 @@ public String getTableName(@NotNull TableName tableName) {
 
     // Synchronization settings
     @YamlComment("Synchronization settings")
+    @YamlKey("synchronization.server_id")
+    public String serverID = "node-1";
```

Comment:
As mentioned, probably best have DataServerID save this in its own YamlDocument-based file, as this breaks users managing their config with symlinks